### PR TITLE
Edited Standard Deviation functions in EggBio

### PR DESCRIPTION
**Pull request by lorijones**

### CODE REVIEW

File: FluEgggui.m [hunk 1]
```diff
@@ -374,53 +374,33 @@
...
-                end
-                if time(tt)/3600<1
-                    RhoeStd = 8.7916;%Kg/m^3 at 22C
-                else
-                    RhoeStd = 1.7663;%Kg/m^3 at 22C
-                end
+                     %% Diameter
+                    DiamStd = 0.4759.*exp(-time/14150)+0.4586;
+                    %% Density
+                    RhoeStd = 19.28.*exp(-timeGC./1973)+1.029;
```
Comment (tgarciabotero) [inline FluEgggui.m]: In line 393: 
RhoeStd = 19.28.*exp(-timeGC./1973)+1.029;
it should be time instead timeGC

File: FluEgggui.m [hunk 2]
```diff
@@ -374,53 +374,33 @@
...
-                if time(tt)/3600<1
-                    RhoeStd = 8.2231;%Kg/m^3 at 22C
-                else
-                    RhoeStd = 0.7780;%Kg/m^3 at 22C
-                end
-                %%
+                    %% Diameter
+                    DiamStd = 0.2631.*exp(-time/22410)+0.3073;
+                	%% Density
+                    RhoeStd = 22.4.*exp(-time/1894)+0.4103;
```
Comment (tgarciabotero) [inline FluEgggui.m:384]: Lori, can you please verify the equation of Density STD?? it does not match with the ppt

File: FluEgggui.m [hunk 3]
```diff
@@ -374,53 +374,33 @@
...
+                     %% Diameter
+                    DiamStd = 0.4759.*exp(-time/14150)+0.4586;
+                    %% Density
+                    RhoeStd = 19.28.*exp(-timeGC./1973)+1.029;
             end % Species selection
             %% ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
             %% Diameter fit + scatter
             Dvar(tt,1) = single(normrnd(D(tt),DiamStd));
-            while (Dvar(tt)>=D(tt)+DiamStd)||(Dvar(tt)<=D(tt)-DiamStd)
+            while (Dvar(tt)>=D(tt)+2*DiamStd)||(Dvar(tt)<=D(tt)-2*DiamStd)
```
Comment (tgarciabotero) [inline FluEgggui.m]: Lori, would you mind adding a line of code explaining that we are using a random variable with 2 time the standard deviation as a limit and tut your initials in parenthesis??
thanks

File: FluEgggui.m [hunk 4]
```diff
@@ -374,53 +374,33 @@
             %% ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
             if strcmp(specie,'Silver')%if specie=='Silver'
                 %% STD
```
Comment (tgarciabotero) [inline FluEgggui.m:380]: Lori, can you please add a small line explaining how did you calculated this function and put your initials at the end of the comment??

Thanks.

**Review by tgarciabotero — COMMENTED** [review]
Hi Lori,
Thank you for improving the FluEgg Bio function. I think this is a great additions.
Can you please address this minor comments please?

File: FluEgggui.m (hunk 4 above)
Comment (lorijones) [reply FluEgggui.m:380]: In the new pull request, this comment is added.

File: FluEgggui.m (hunk 2 above)
Comment (lorijones) [reply FluEgggui.m:384]: It was written incorrectly in the ppt. This is the correct function.

File: FluEgggui.m (hunk 1 above)
Comment (lorijones) [reply FluEgggui.m]: This is fixed in new pull request.

File: FluEgggui.m (hunk 3 above)
Comment (lorijones) [reply FluEgggui.m]: This comment is also added in new pull request.

**Review by tgarciabotero — APPROVED** [review]

**Review by tgarciabotero — COMMENTED** [review]
Great job Lori, Thank you

**Pull request merged**